### PR TITLE
EXP: updating package imports

### PR DESCRIPTION
**Pull request by sosey**
Not saying this is the way to go, and I haven't run the test suite, but these changes build in py2 and py3 and allow doc creation in my setup with the current master. Check if it works for you and in rtd.

### CODE REVIEW

**Comment by sosey** [comment]
http://drizzlepac-sosey.readthedocs.io/

**Comment by mcara** [comment]
@sosey 👏

**Comment by mcara** [comment]
I see no harm in trying this in dev.

File: drizzlepac/astrodrizzle.py [hunk 1]
```diff
@@ -40,7 +40,6 @@
 from stsci.tools import teal, logutil, textutil
 
 from . import adrizzle
-from . import ablot
```
Comment (mcara) [inline drizzlepac/astrodrizzle.py]: @sosey This crashes the code. Why did you remove this import? I am making a PR to restore this one import.

**Comment by jhunkeler** [comment]
@mcara to prevent this "oh no" type stuff in the future you can test her changes before merging to master.
```
$ cd drizzlepac

# Add @sosey's repo as a remote
$ git remote add sosey https://github.com/sosey/drizzlepac.git

# Checkout @sosey's PR in a detached HEAD
$ git checkout sosey/whatever-her-branch-is-called

$ python setup.py [whatever]
# review & test & tweak things

# Contribute back to the PR
$ git checkout -b my-tweaks
$ git add your_changes
$ git commit -m 'tweaks and whatnot'

# Then open a PR against @sosey's branch on Github
```

**Comment by mcara** [comment]
@jhunkeler Will do...

**Comment by sosey** [comment]
ooh, yah, I didn't really expect my changes to be merged at all...sorry

**Comment by sosey** [comment]
FWIW - RTD is still showing the deployment version by default, it should be changed to latest or master through the RTD settings for the package.

**Comment by sosey** [comment]
I went ahead and updated RTD to show the latest build since nadia is away
